source/install.txt

```diff
@@ -17,6 +17,8 @@ Prerequisites
 
 .. include:: /includes/fact-connect-prereqs.rst
 
```

Comment:
(question) 
Where are folks going to get the information about what OS versions (etc) are supported? Perhaps add a link to wherever that is?